training/utils.py

```diff
@@ -74,7 +74,7 @@ def get_one_hot_encoded_verse(surah_num, ayah_num):
     return decoder_input, decoder_target
 
 
-def shuffle_together(array1, array2, array3):
+def shuffle_together(array1, array2, array3, array4):
     """
     A helper method to randomly shuffle the order of three arrays while keeping their relative orders the same.
```

Comment:
update comment: order of *three -> four* arrays

We should consider writing a function that just takes arbitrary number of arrays using `*args` (incase some old code references the three array version)

Reply:
Done.